sdk/test-utils/perfstress/package.json

```diff
@@ -68,6 +68,7 @@
     "@types/minimist": "~1.2.0"
   },
   "devDependencies": {
+    "@azure/core-client":"^1.3.1",
```

Comment:
@HarshaNalluru: Should this be a dev dep or a full dep?  I think dev dep is more appropriate, since it's only used by the `ServiceClientGet` sample test, not the perf framework itself.

Reply:
Sounds good.